website/docs/index.html.markdown

```diff
@@ -192,6 +192,30 @@ resource "vcd_network_routed" "net1" {
 
 Note that when connecting with API tokens you can't create or modify users, roles, global roles, or rights bundles.
 
+### Connecting with a Service Account API token
+
+With VCD 10.4.0+, you can connect using a service account API token, as defined in the [documentation](https://blogs.vmware.com/cloudprovider/2022/07/cloud-director-service-accounts.html). Because a new API token is provided on every authentication request, the user is required to provide a file in `json` format with the current API key. e.g:
+```json
+{"refresh_token":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"}
+```
+
+~> **NOTE:** The service account needs to be in `Active Stage` and it's up to the user to provide the initial API token.
+
+```hcl
+provider "vcd" {
+  user                 = "none"
+  password             = "none"
+  auth_type            = "service_account"
+  api_token            = "token.json"
```

Comment:
Probably it's me and Friday, but shouldn't the example include `service_account_token_file` parameter when `auth_type "service_account"`?

Reference:
https://github.com/vmware/terraform-provider-vcd/pull/1040/files#diff-e96a5cf811e4de4a40ed325399541cc28aef6972ab94e260331cb58898cd6a8bR366-R369

Reply:
Fixed.